main.tf

```diff
@@ -7,6 +7,7 @@ resource "github_membership" "maintainers" {
 resource "github_team_membership" "maintainers" {
   for_each = toset(compact(distinct(concat(var.maintainers, var.board))))
   username = each.key
+  role     = "maintainer"
```

Comment:
This was defaulting to member, which is wrong for a maintainer group 😆

Reply:
Maintainer lets them change the group settings remember